### ISSUE TITLE
Cargo check fails when compiling Yew with web_sys feature

### ISSUE THREAD

**Issue by RyanBluth** [rust-analyzer/rust-analyzer#3592]
`cargo check` fails building yew with the web_sys feature enabled. This is because the target needs to be set to `wasm32-unknown-unknown` using `cargo check --target wasm32-unknown-unknown`.  This prevents rust analyzer from reporting the results of cargo check. It would be nice if there were a way to configure the cargo check target using VS Code's work space settings.

You can test this by cloning https://github.com/yewstack/yew-wasm-pack-template

**Comment by kiljacken** [comment]
You should "just" be able to add the --target parameter if you make sure
"all targets" is disabled for cargo check

**Comment by RyanBluth** [comment]
Like so? 
![screenshot](https://user-images.githubusercontent.com/5282031/76694364-12f5d280-6648-11ea-9c08-cde2b9bcbcc1.png)

I tried that before creating the issue and it doesn't seem to solve the problem.

 I also tried creating a .cargo folder with a config file specifying the build target as explained here: https://doc.rust-lang.org/cargo/reference/config.html. This allows me to run `cargo check` successfully but doesn't seem to solve the problem with rust-analyzer
